code/modules/antagonists/heretic/status_effects/void_chill.dm

```diff
@@ -0,0 +1,101 @@
+/*!
+ * Contains the "Void Chill" status effect. Harmful debuff which freezes and slows down non-heretics
+ * Cannot affect silicons (How are you gonna freeze a robot?)
+ */
+/datum/status_effect/void_chill
+	id = "void_chill"
+	duration = 30 SECONDS
+	alert_type = /atom/movable/screen/alert/status_effect/void_chill
+	status_type = STATUS_EFFECT_REFRESH //Custom code
+	on_remove_on_mob_delete = TRUE
+	remove_on_fullheal = TRUE
+	///Current amount of stacks we have
+	var/stacks
```

Comment:
We have a whole subtype of status effect with a stacks api, `/datum/status_effect/stacking`, why not use it?

It does a lot of what this does

Reply:
I tried looking into making it a stacking subtype and it appears this is meant for things with an overlay that gains as you gain stacks.

But due to its nature this status effect doesn't really do that with overlays/underlays so it runtimes a lot. It also doesn't really support what I want exactly.